examples/async_transcribe_local_media_file.js

```diff
@@ -4,7 +4,8 @@ const token = require('./config/config.json').access_token;
 
 (async () => {
     // Initialize your client with your Rev AI access token
-    var client = new revai.RevAiApiClient(token);
+    // Make sure the base url matches the deployment for your Rev AI access token
+    var client = new revai.RevAiApiClient(accessToken: token, version: 'v1', baseUrl: BaseUrl.US);
```

Comment:
did we hide the base url intentionally before this change?

Reply:
im not sure intentionally, there was only 1 base url so there was no need to offer the option to set it I guess @amikofalvy

Reply:
When we created this SDK originally there was no need to allow the user to change the base url so we did not expose it as part of the interface.

Interestingly, we did want this functionality for our integration tests so that we could run them against non production environments:

https://github.com/revdotcom/revai-node-sdk/blob/8bca62539300fdfb05d1ee7086207b7a0cd05597/test/integration/src/client-helper.js#L12-L16

Reply:
Yes, I was wondering what was the purpose when tried to test the client across multiple environments.
We agreed to introduce a new constructor that will stay fully compatible with the existing code, but is more extensible.

Example:
https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgMIHsQ2Ac2QbwChkTkx0BrCEAfgC5kBnMKUHAbmNIDdpHhM9JizadSyALZwcwBEJABXCQCNonAL6FCCADZxGjNDuDUwBLiQAOrbnEjIA+gkzYcDDFlzIAvAXWcLZGcQZigFBHIoAApg13cXLwAfYVYQHAAaZF4ofkwGULYfZABybgBGYoBKQKJxcWAYZCiwAE9LCHRG2K9vXpL0ZQArCAiq8zqJsgALYEYAOicEvF8AeSGRsDn9fhwQZpn5xc8MoKXkfTQlyrFJ8TADhe6cOezckGQaGl9XgRAbifUyAgOkYKFqt1I91mjyWc3IVHevie-whUMOTxefF+RR+mBR4k0BK04msoDAUUq40mwUY6B0EDmOnQOH20KOrmugU0mkIAG1AiAIAB3IwmcBRfBkSjUBjFMoAJgAzMVkOpKukBcLRaYosV4dQFcr1ZqRahjDq9dKQIbiplSvKqoQALpzGDoKAAUUQUyiKG8AD4gXNSeLKpzCEA